### PR TITLE
Add some misc recipies

### PR DESCRIPTION
**Pull request by Pdiddy973**
adds crafting recipe to flesh block
adds getting brick for crushing bricks
adds getting rotten flesh form crushing flesh block
adds getting magma cream from crushing magma block

### CODE REVIEW

**Comment by Pdiddy973** [comment]
also added Certus and Spider bee

File: config/resourcefulbees/bees/bees_gem/bred/Certus.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    "primaryColor": "#d2d6f7",
+    "secondaryColor": "#303030"
+  },
+  "MutationData": {
+    "hasMutation": false
+  },
+  "CentrifugeData": {
+    "hasCentrifugeOutput": true,
+    "mainOutput": "appliedenergistics2:certus_quartz_crystal",
+    "mainOutputCount": 2
```
Comment (mkaito) [inline config/resourcefulbees/bees/bees_gem/bred/Certus.json:17]: 2 per comb seems a little excessive.

File: config/resourcefulbees/bees/bees_gem/bred/Certus.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    "isBreedable": true,
+    "parent1": "Crystal",
+    "parent2": "Diamond",
+    "feedItem": "appliedenergistics2:certus_quartz_crystal",
+    "feedAmount": 4
+  },
+  "TraitData": {
+    "hasTraits": false
+  }
+}
```
Comment (mkaito) [inline config/resourcefulbees/bees/bees_gem/bred/Certus.json:32]: Could you add a trailing newline?

File: config/resourcefulbees/bees/bees_misc/natural/RGBee.json [hunk 3]
```diff
@@ -2,13 +2,12 @@
...
   "hasHoneycomb": true,
-  "ColorData": {
-    "isBeeColored": true,
-    "primaryColor": "#ffffff",
-    "honeycombColor": "#ffffff",
-    "isRainbowBee": true,
-    "isGlowing": true
-  },
+  "ColorData": {  
+  "isBeeColored": true,
```
Comment (mkaito) [inline config/resourcefulbees/bees/bees_misc/natural/RGBee.json:6]: indentation

File: config/resourcefulbees/bees/bees_mob/natural/Spider.json [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+    "secondaryColor": "#910000"
+  },
+  "MutationData": {
+    "hasMutation": false
+  },
+  "CentrifugeData": {
+    "hasCentrifugeOutput": true,
+    "mainOutput": "minecraft:string",
+	"mainOutputCount": 1,
+	"secondaryOutput" : "xreliquary:chelicerae",
```
Comment (mkaito) [inline config/resourcefulbees/bees/bees_mob/natural/Spider.json:18]: indentation

File: config/resourcefulbees/bees/bees_mob/natural/Spider.json [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+  },
+  "BreedData": {
+    "isBreedable": true,
+    "feedItem": "xreliquary:chelicerae",
+    "feedAmount": 3
+  },
+  "TraitData": {
+    "hasTraits": false
+  }
+}
```
Comment (mkaito) [inline config/resourcefulbees/bees/bees_mob/natural/Spider.json:35]: trailing newline

File: config/resourcefulbees/resources/assets/resourcefulbees/lang/en_us.json [hunk 6]
```diff
@@ -226,5 +226,13 @@
...
+    "entity.resourcefulbees.lava_bee" : "Lava Bee",
+    "block.resourcefulbees.certus_honeycomb_block": "Certus Honeycomb Block",
+    "item.resourcefulbees.certus_honeycomb": "Certus Honeycomb",
+    "item.resourcefulbees.certus_bee_spawn_egg": "Certus Bee Spawn Egg",
+    "entity.resourcefulbees.certus_bee": "Certus Bee",
+	"block.resourcefulbees.spider_honeycomb_block": "Spider Honeycomb Block",
+    "item.resourcefulbees.spider_honeycomb": "Spider Honeycomb",
+    "item.resourcefulbees.spider_bee_spawn_egg": "Spider Bee Spawn Egg",
+    "entity.resourcefulbees.spider_bee": "Spider Bee"
 }
```
Comment (mkaito) [inline config/resourcefulbees/resources/assets/resourcefulbees/lang/en_us.json:238]: newline

File: config/resourcefulbees/resources/assets/resourcefulbees/lang/en_us.json [hunk 7]
```diff
@@ -226,5 +226,13 @@
     "block.resourcefulbees.lava_honeycomb_block" : "Lava Honeycomb Block",
     "item.resourcefulbees.lava_honeycomb" : "Lava Honeycomb",
     "item.resourcefulbees.lava_bee_spawn_egg" : "Lava Bee Spawn Egg",
-    "entity.resourcefulbees.lava_bee" : "Lava Bee"
+    "entity.resourcefulbees.lava_bee" : "Lava Bee",
+    "block.resourcefulbees.certus_honeycomb_block": "Certus Honeycomb Block",
+    "item.resourcefulbees.certus_honeycomb": "Certus Honeycomb",
+    "item.resourcefulbees.certus_bee_spawn_egg": "Certus Bee Spawn Egg",
+    "entity.resourcefulbees.certus_bee": "Certus Bee",
```
Comment (mkaito) [inline config/resourcefulbees/resources/assets/resourcefulbees/lang/en_us.json:233]: Certus _Quartz_ ?

**Review by mkaito — CHANGES_REQUESTED** [review]
You can fix most of this by just running your files through a formatter such as https://beautifier.io/ or https://prettier.io/

File: config/resourcefulbees/resources/assets/resourcefulbees/lang/en_us.json [hunk 8]
```diff
@@ -226,5 +226,13 @@
...
     "item.resourcefulbees.lava_bee_spawn_egg" : "Lava Bee Spawn Egg",
-    "entity.resourcefulbees.lava_bee" : "Lava Bee"
-}
\ No newline at end of file
+    "entity.resourcefulbees.lava_bee" : "Lava Bee",
+    "block.resourcefulbees.certus_honeycomb_block": "Certus Honeycomb Block",
+    "item.resourcefulbees.certus_honeycomb": "Certus Honeycomb",
+    "item.resourcefulbees.certus_bee_spawn_egg": "Certus Bee Spawn Egg",
+    "entity.resourcefulbees.certus_bee": "Certus Bee",
+	"block.resourcefulbees.spider_honeycomb_block": "Spider Honeycomb Block",
```
Comment (mkaito) [inline config/resourcefulbees/resources/assets/resourcefulbees/lang/en_us.json:234]: indentation

File: config/resourcefulbees/bees/bees_mob/natural/Spider.json [hunk 9]
```diff
@@ -0,0 +1,35 @@
...
+      "primaryColor": "#030303",
+      "secondaryColor": "#910000"
+  },
+  "MutationData": {
+      "hasMutation": false
+  },
+  "CentrifugeData": {
+      "hasCentrifugeOutput": true,
+      "mainOutput": "minecraft:string",
+      "mainOutputCount": 1,
```
Comment (mkaito) [inline config/resourcefulbees/bees/bees_mob/natural/Spider.json:17]: 1 string per comb seems a little low. How about 4?

**Review by mkaito — CHANGES_REQUESTED** [review]

File: config/resourcefulbees/bees/bees_gem/bred/Fluix.json [hunk 10]
```diff
@@ -0,0 +1,36 @@
...
+        "isBreedable": true,
+        "parent1": "Certus",
+        "parent2": "Redstone",
+        "feedItem": "appliedenergistics2:purified_fluix_crystal",
+        "feedAmount": 4
+    },
+    "TraitData": {
+        "hasTraits": false
+    }
+}
```
Comment (mkaito) [inline config/resourcefulbees/bees/bees_gem/bred/Fluix.json:36]: We discussed briefly in dev chat, and the few of us present agreed that we do not want a bee for a composite type like Fluix. We defer to @WhitePhant0m though.

Comment (mkaito) [reply config/resourcefulbees/bees/bees_gem/bred/Fluix.json:36]: Asked Phant0m, he said nay.

Comment (WhitePhant0m) [reply config/resourcefulbees/bees/bees_gem/bred/Fluix.json:36]: Yeah, I don't think a fluix bee would be good, and I will probably be nuking the fluix plant too.

Comment (Pdiddy973) [reply config/resourcefulbees/bees/bees_gem/bred/Fluix.json:36]: Fair enough, made the fluix bee since we had the fluix crop.

**Review by mkaito — APPROVED** [review]
Thanks!

**Review by mkaito — CHANGES_REQUESTED** [review]
Please split the builder changes to their own PR.

**Comment by Pdiddy973** [comment]
closing, made new PRs that are less messy

**Pull request closed without merging**